src/apps/loyalty/server/route_handlers.tsx

```diff
@@ -0,0 +1,5 @@
+import { NextFunction, Request, Response } from "express"
+
+export function Home(req: Request, res: Response, next: NextFunction) {
```

Comment:
Definitely. 🌞 

The reason I started with this really simple example, is that I had an issue with even import'ing the app, it was choking on some Isomorphic Relay stuff (Container not found).

At some point we'll have to figure that out, but that's sort of why I was just trying something simple to start.